substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/ResourceAttributes.java

```diff
@@ -0,0 +1,75 @@
+package com.oracle.svm.core.jdk;
+
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileTime;
+import java.util.Formatter;
+
+public class ResourceAttributes implements BasicFileAttributes {
+
+    private final ResourceFileSystem.Entry entry;
+
+    public ResourceAttributes(ResourceFileSystem.Entry entry) {
+        this.entry = entry;
+    }
+
+    @Override
+    public FileTime lastModifiedTime() {
+        return null;
+    }
+
+    @Override
+    public FileTime lastAccessTime() {
+        return null;
+    }
+
+    @Override
+    public FileTime creationTime() {
+        return null;
+    }
+
+    @Override
+    public boolean isRegularFile() {
+        return !entry.isDirectory();
+    }
+
+    @Override
+    public boolean isDirectory() {
+        return entry.isDirectory();
+    }
+
+    @Override
+    public boolean isSymbolicLink() {
+        return false;
+    }
+
+    @Override
+    public boolean isOther() {
+        return false;
+    }
+
+    @Override
+    public long size() {
+        return entry.size();
+    }
+
+    @Override
+    public Object fileKey() {
+        return null;
+    }
+
+    public String toString() {
+        StringBuilder sb = new StringBuilder(1024);
+        Formatter fm = new Formatter(sb);
+        fm.format("    creationTime    : null%n");
```

Comment:
What will Java return here?

Reply:
In current implementation the creationTime, lastAccessTime, lastModifiedTime is not supported (returning null). We will need to support that as well, I'll add TODO.

Reply:
yup, just do the same thing Java would return here.

Reply:
Done.